packages/material-ui/src/AvatarGroup/AvatarGroup.js

```diff
@@ -2,36 +2,74 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { isFragment } from 'react-is';
 import clsx from 'clsx';
-import { chainPropTypes } from '@material-ui/utils';
-import { withStyles } from '../styles';
+import { chainPropTypes, deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Avatar from '../Avatar';
+import avatarGroupClasses, { getAvatarGroupUtilityClass } from './avatarGroupClasses';
 
 const SPACINGS = {
   small: -16,
   medium: null,
 };
 
-export const styles = (theme) => ({
+const overridesResolver = (props, styles) => {
+  return deepmerge(styles.root || {}, {
+    [`& .${avatarGroupClasses.avatar}`]: styles.avatar,
+  });
+};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes } = styleProps;
+
+  const slots = {
+    root: ['root'],
+    avatar: ['avatar'],
+  };
+
+  return composeClasses(slots, getAvatarGroupUtilityClass, classes);
+};
+
+const AvatarGroupRoot = experimentalStyled(
+  'div',
+  {},
+  {
+    name: 'MuiAvatarGroup',
+    slot: 'Root',
+    overridesResolver,
+  },
+)({
```

Comment:
```suggestion
)(({ theme }) => ({
  [`& .MuiAvatar-root`]: {
    border: `2px solid ${theme.palette.background.default}`,
    boxSizing: 'content-box',
    marginLeft: -8,
    '&:last-child': {
      marginLeft: 0,	
    },	
  },
```

We need to add these overrides as overrides for the `Avatar` component as well, so that they will affect the children Avatar too, not just the extra avatar. With this change I believe we will be able to fix the screen regression we have now on the PR:

![image](https://user-images.githubusercontent.com/4512430/104893388-a89e7480-5973-11eb-9a52-fd39526561a0.png)